pdokservicesplugin/pdokservicesplugin.py

```diff
@@ -600,7 +600,7 @@ def create_new_layer(self):
             return QgsVectorLayer(uri, title, "OAPIF")
         elif servicetype == "api tiles":  # OGC API Tiles
 
-            # CRS met oat werkt nog niet correct in qgis/gdal
+            # CRS does not work correctly in qgis/gdal. We can set a crs (non-webmercator), but it is not displayed as expected.
```

Comment:
Ik zou hiervan maken: not yet implemented with other projections. of not expected. 
```suggestion
            # CRS does not work as expected in qgis/gdal. We can set a crs (non-webmercator), but it is rendered elsewhere.
```